lib/Convos.pm

```diff
@@ -138,14 +139,39 @@ sub _assets {
     ->design({
     desktop_browser => {},
     android_chrome  => {
-      picture_aspect => "shadow",
-      manifest       => {name => "Convos", display => "standalone", orientation => "portrait"},
-      theme_color    => "#00451D"
+      picture_aspect => 'shadow',
+      manifest       => {name => 'Convos', display => 'standalone', orientation => 'portrait'},
+      theme_color    => '#00451D'
     },
+    firefox_app => {
+        picture_aspect=> 'circle',
+        keep_picture_in_circle=> 'true',
```

Comment:
Looks like this part isn't tidied.

Another thing not related to Convos: Should there be a config file that AssetPack::Pipe::Favicon looks up, so we don't need all of this inside the application?